lib/ui/page/home/page/chat/view.dart

```diff
@@ -829,13 +819,12 @@ class ChatView extends StatelessWidget {
                 context,
                 c,
                 item: element,
-                overlay:
-                    e.value.author.id != c.me || element is ChatInfoElement,
                 child: ChatItemWidget(
                   chat: c.chat!.chat,
                   item: e,
                   me: c.me!,
                   avatar: !previousSame,
+                  appendAvatarPadding: !c.selecting.value,
```

Comment:
I've recorded an animation of toggling selection mode on and off in slow motion (0.5x speed). Notice the message jumping rapidly when toggling selection on. It doesn't look good, don't you think? Can't this be fixed?


https://github.com/user-attachments/assets/263a6dda-e08e-412f-ba8f-2261143e7e4a